### ISSUE TITLE
Localize the locator overlay to the extent possible

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#7737]
The new Mapbox locator overlay is great, but it currently shows labels only in their local language(s). It'd be better to request tiles in the user's language if it's one that's supported by Mapbox. Cc: @mikelmaron

<img width="949" alt="Screen Shot 2020-06-25 at 11 24 17 AM" src="https://user-images.githubusercontent.com/2046746/85749069-86db5880-b6d6-11ea-8f24-7f3b57b752bc.png">

![image](https://user-images.githubusercontent.com/2046746/85749476-e174b480-b6d6-11ea-9ace-c70050cb00f6.png)

**Comment by mikelmaron** [comment]
@quincylvania there's a couple ways this could be achieved. we could create different styles for each language, and iD would request the appropriate one for the users environment. or iD could use vector rather than raster tiles, and modify the styling on the front end; that is simple to code, but would mean dependency on a library to handle vector tiles.

**Comment by quincylvania** [comment]
@mikelmaron Hmm I'd rather not add a vector tile library just for this. Vector tiles are probably something iD should have first-class support for eventually. You can sort of load an vector tile endpoint as the custom map data layer, but the support is minimal and bug-prone.

For now I think it's fine to swap between language-specific styles. Though I'm assuming it's tough to keep them in sync, so this can wait until everyone's satisfied with the styling.

**Comment by mikelmaron** [comment]
Yes makes sense to launch the new style, make sure there's no issues with the styling, and then look at cloning and tweaking for different languages.

In parallel, presume this wouldn't be too hard to implement in iD. However I'm not clear how we'd add this to the ELI in a way that wouldn't explode the number of overlay layers available.

**Comment by quincylvania** [comment]
A couple remaining issues with the styling, some natural feature labels are either light on light or dark on dark. I think all the labels either need a dark halo with white text or the reverse.

<img width="385" alt="Screen Shot 2021-01-05 at 10 57 15 AM" src="https://user-images.githubusercontent.com/2046746/103668583-4130ff80-4f45-11eb-8595-a6fed4eef77c.png">
<img width="494" alt="Screen Shot 2021-01-05 at 10 56 13 AM" src="https://user-images.githubusercontent.com/2046746/103668585-41c99600-4f45-11eb-98a9-fe361f6cf1f7.png">
